asn1ate/sema.py

```diff
@@ -270,6 +270,8 @@ def __init__(self, elements):
             self.tag_default = TagImplicity.EXPLICIT
 
         self.assignments = [_create_sema_node(token) for token in assignments.elements]
+	self.imports = _create_sema_node(imports)
```

Comment:
I've done that, indeed based on just adding self.imports and self.exports and let __str__ do the work.  Tested by printing the modules, which worked and did include IMPORTS and EXPORTS where deemed useful.